functions/index.js

```diff
@@ -159,9 +159,16 @@ exports.memberTagsHandler = functions.handler.firestore.document
       const prevDoc = event && event.before && event.before.data();
       const newDoc = event && event.after && event.after.data();
 
+      // Check if we are dealing with a multidimensional array
+      const isMultidimensional = (tag) =>
+        typeof tag === "object" && typeof tag.path !== "undefined" && typeof tag.key !== "undefined"
+
       // Retrieves subscriber tags before/after write event
       const getTagsFromEventSnapshot = snapshot => tagsConfig.memberTags.reduce((acc, tag) => {
-        const tags = _.get(snapshot, tag);
+        const tags = isMultidimensional(tag)
```

Comment:
Please update to

```
const tags = isObjectConfig(tag)
          ? _.get(snapshot, tag.tagPath).map((parent) => typeof parent === "object" && typeof tag.keySelector !== "undefined" ? _.get(parent, tag.keySelector) : parent)
          : _.get(snapshot, tag);
```

Reply:
Please extend on this to check that the return value when using tagPath is an array before calling map